### PR TITLE
bitlbee: update to @3.6.1

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update to @3.6.1, change master_sites to Github.

###### Type(s)

- [ ] bugfix
- [x ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

### CODE REVIEW

File: irc/bitlbee/Portfile [hunk 1]
```diff
@@ -16,10 +16,12 @@ long_description    bitlbee is an IRC server that can connect to other IM \
                     instant messaging.
 
 homepage            https://www.bitlbee.org/
-master_sites        https://get.bitlbee.org/src/
+master_sites        https://github.com/bitlbee/bitlbee/archive/refs/tags/
```
Comment (reneeotten) [inline irc/bitlbee/Portfile:19]: if you want to do this you should instead use the [`github` PG](https://guide.macports.org/index.html#reference.portgroup.github) which will do all kinds of things for you automatically.

File: irc/bitlbee/Portfile [hunk 2]
```diff
@@ -29,11 +31,11 @@ depends_lib         port:gettext \
                     port:libgpg-error \
                     port:libiconv
 
-patchfiles          patch-configure.diff \
-                    patch-Makefiles.diff
+patchfiles          patch-configure.diff
 
 # Calls ld directly which makes universal hard
-universal_variant   no
+# universal_variant   no
```
Comment (reneeotten) [inline irc/bitlbee/Portfile:37]: please don't comment out lines if not needed anymore, just remove them

File: irc/bitlbee/Portfile [hunk 3]
```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                bitlbee
-version             3.5.1
-revision            1
+version             3.6.1
+revision            0
 categories          irc net
 platforms           darwin
```
Comment (reneeotten) [inline irc/bitlbee/Portfile:9]: this is the default value for `platforms` since MacPorts v2.7 so the line can/should be removed

**Review by reneeotten — CHANGES_REQUESTED** [review]
a few comments below and I notice that GitHub thinks that there are conflicting files, so you should probably `rebase` with "master" and resolve any possible conflicts it might tell you about

**Comment by barracuda156** [comment]
> a few comments below and I notice that GitHub thinks that there are conflicting files, so you should probably `rebase` with "master" and resolve any possible conflicts it might tell you about

@reneeotten The port was updated by @ryandesign some hours before and that was not yet in the master, so I did not see any changes. It looks like he used an earlier version, so it may be worth updating it still.
But I think he should decide that.

**Comment by barracuda156** [comment]
> the port is still `nomaintainer` so you can certainly feel free to update the port; you can do that in this PR. Just rebase your branch with `master`, do the update and then force-push to this PR.

@reneeotten Alright, I can do that tomorrow once back to workstation.

**Comment by barracuda156** [comment]
@ryandesign Thank you for clarification! Surely, no need to use 3.6-1 then.

**Pull request closed without merging**